packages/preview/cetz-0.0.1/typst.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "cetz"
+version = "0.0.1"
+repository = "https://github.com/johannes-wolf/typst-canvas"
+entrypoint = "lib.typ"
+authors = [
+  "Johannes Wolf <https://github.com/johannes-wolf>",
+  "fenjalien <https://github.com/fenjalien>"
+]
+license = "Apache-2.0"
+description = "Drawing with Typst made easy, providing an API inspired by TikZ and Processing."
+exclude = [ "*.png", "manual.pdf", "manual.typ" ]
```

Comment:
is it intentional that `gallery` isn't excluded completely? the three Typst files aren't big, but maybe a bit pointless.

Reply:
Done.